backend/src/main/java/org/eclipse/tractusx/semantics/hub/bamm/BammHelper.java

```diff
@@ -154,4 +157,31 @@ public Try<String> getExamplePayloadJson(Aspect aspect) {
 
         return Try.of(payloadGenerator::generateJson);
     }
+
+    public Try getAasSubmodelTemplate(Aspect aspect, String aasFormat) {
+        AspectModelAASGenerator aasGenerator = new AspectModelAASGenerator();
+        ByteArrayOutputStream stream = new ByteArrayOutputStream();
+        
+        try {
+            switch(aasFormat) {
+                case "file":
+                    aasGenerator.generateAASXFile(aspect, (String s) -> {
+                        return stream;
+                    });
+                    stream.close();
+                    return Try.of(stream::toByteArray);
+                case "xml":
+                    aasGenerator.generateAasXmlFile(aspect, (String s) -> {
+                        return stream;
+                    });
+                    stream.close();
+                    return Try.of(stream::toString);
+                default:
+                    return Try.failure(new Exception("Wrong AAS output format."));
```

Comment:
To improve the error output it makes sense for me to add the `aasFormat` to the exception message.

`new Exception(String.format("Wrong AAS output format %s", aasFormat))`